docs/src/admin-manual/dashboard-config.md

```diff
@@ -0,0 +1,55 @@
+# Dashboard configuration
+
+## Institution logo
+
+The Enduro dashboard can display an institutional logo in the page header by
+setting the following [Vite environment variables].
+
+When running an Enduro Dashboard Docker container, such as when using the Tilt
+development environment, the Vite environment variables can be set via the
+`dashboard/.env` (production), `dashboard/.env.development` (development), or
+`dashboard/.env.test` (test) environment files.
+
+If the Enduro Dashboard is not being run via Docker, but the code is being built
+with [Vite] then the institution logo can be configured in the production
+environment file (`dashboard/.env`) before building the distribution files.
+
```

Comment:
So nice that you added this! It would be great to mention the replacement script we use to be able to set those env. vars. after build (needed when using RPM packages for example).

https://github.com/artefactual-sdps/enduro/blob/main/dashboard/hack/inject-vite-envs.sh

We already mention this script in the IAC page, where we show the other dashboard env. vars. for OIDC configuration. It would be nice to organize this a bit better, not sure exactly how and it doesn't need to happen here.

Reply:
Another idea would be to add this documentation to the general configuration docs I am already working on. As it is, this PR's changes to the README toc will conflict with my PR, and even if kept apart, both pages should be updated to point to each other as "see also" for other configuration file stuff.

Just a thought!

Reply:
@jraddaoui I think the variable injection script is only necessary for changing the variables after the Vite distribution files are built.  I think for most institutions changing the logo variables in the config files then building the distribution files will be fine because the logo values shouldn't change often. 

I can still add the note about the injection script if you feel it's necessary but it's fairly complex and I'd rather avoid that complexity if possible.